chef.gemspec

```diff
@@ -38,6 +38,8 @@ Gem::Specification.new do |s|
   s.add_dependency "diff-lcs", "~> 1.2", ">= 1.2.4"
   s.add_dependency "ffi-libarchive"
   s.add_dependency "chef-zero", ">= 14.0.11"
+  s.add_dependency "train-core"
```

Comment:
Do we want to keep this since we bring this in already via chef-core?

Reply:
I could go any way. I don't know of any problem caused by leaving it in, or taking it out.

When we change this implementation to use the same library that is chef-core today, we would surely remove this. But on its own, presuming that we'll get a dependency we require from another dependency doesn't feel right.